CHANGELOG.md

```diff
@@ -9,6 +9,12 @@ Canonical reference for changes, improvements, and bugfixes for Boundary.
 * Adds support for Azure Virtual Machine Scale Sets in the Azure plugin
   ([PR](https://github.com/hashicorp/boundary-plugin-azure/pull/22)).
 
+### Bug fixes
+
+* Fixed an issue where closing a channel failed to exit a loop, which would
+  cause a massive spike in CPU usage over time. This change only affects
+  Enterprise.
```

Comment:
I know we're intentionally vague, but maybe we should say that this affects the worker only?
```suggestion
* Fixed an issue in the worker where closing a channel failed to exit a loop, which would
  cause a massive spike in CPU usage over time. This change only affects
  Enterprise.
```

Reply:
Good point, I will apply this change.